docker/package/packages.py

```diff
@@ -411,6 +411,12 @@ def mk_node_unit(
         )
     )
 
+packages.append(
```

Comment:
`TezosSaplingParamsPackage` was removed from `packages` on purpose to avoid re-publishing it by CI, since it isn't supposed to be updated. If you want to have a way to build it, I propose to add a conditional for it

Reply:
Makes sense, I add a boolean flag for this. It's disabled by default.

Reply:
Note that there is also `tezos-baking` package, which doesn't actually package any Octez binaries, but packages systemd services and some python scripts. Not sure if you intend to put it into `tezos/tezos` releases as well since this package tends to update multiple times between upstream releases

Reply:
I see that `tezos-baking` is already built.

Reply:
Yep, I don't see a reason against it.